src/app-config/auth.ts

```diff
@@ -32,6 +37,7 @@ export default class AuthClient {
       this.auth_results = JSON.parse(token.password) as AuthResults;
       const expires_at = this.auth_results.issued_at + this.auth_results.expires_in;
       // Refresh the token if its expired to force a docker login
+      // TODO:46:can we refresh a token without the username/password?  We're going to have to if we want a silent refresh...
```

Comment:
@tjhiggins, today we do a silent token refresh with the username/password. I'm digging into ways this might be possible only using the existing token.

Reply:
I've removed this, ignore

Reply:
I assume from our conv on Friday you're investigating using offline-access? To recap I'm okay switching everything to use refresh tokens.